src/current/v22.2/secure-cockroachdb-kubernetes.md

```diff
@@ -512,18 +498,18 @@ If you previously [authenticated with `cockroach cert`](#example-authenticating-
 
     ~~~
     Certificate directory: certs
-      Usage  | Certificate File |    Key File     |  Expires   |                                                                                                                                  Notes                                                                                                                                  | Error
+        Usage  | Certificate File |  Key File   |  Expires   |                                                                  Notes                                                                  | Error
```

Comment:
This output is misaligned, making it a bit hard to read.

Reply:
Removed the output, which is too wide to format well and where the Error column is empty for all rows.